Zappy_IA_Report/NN/Code/Tutorial3.py

```
@@ -0,0 +1,37 @@
+
+predictions = model.predict(test_images)
+
+print(predictions[0])
+print(class_names[np.argmax(predictions[0])])
+
+for i in range(5):
+    plt.grid(False)
+    plt.imshow(test_images[i], cmap=plt.cm.binary)
+    plt.xlabel(class_names[test_labels[i]])
+    plt.title(class_names[np.argmax(predictions[i])])
+    plt.show()
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)